cuppa/src/main/java/org/forgerock/cuppa/model/Options.java

```diff
@@ -75,14 +65,29 @@ public static Options copyOf(Options options) {
     }
 
     /**
+     * Removes an option.
+     *
+     * @param optionClass The class of the option to remove.
+     * @param <O> The type of the option.
+     * @return A new Option instance without the specified option.
+     */
+    public <O extends Option<?>> Options remove(Class<O> optionClass) {
+        Options copy = copyOf(this);
```

Comment:
does `copyOf` still need to be public? I can't think of a reason why you'd want to make a copy as it's immutable.